APIs/APIManager.m.orig

```diff
@@ -0,0 +1,158 @@
+//
+//  APIManager.m
+//  social-maps
```

Comment:
I'm wondering why this file is in here? with the .orig extension? should we delete it?